packages/server/lib/hooks/hooks.ts

```diff
@@ -140,9 +140,16 @@ export const connectionRefreshSuccess = async ({
         connection_id: connection.id
     });
 
-    const slackNotificationService = new SlackService({ orchestratorClient: getOrchestratorClient(), logContextGetter });
+    const slackNotificationService = new SlackService({ orchestrator, logContextGetter });
 
-    await slackNotificationService.removeFailingConnection(connection, connection.connection_id, 'auth', null, environment.id, config.provider);
+    await slackNotificationService.removeFailingConnection({
+        connection,
+        name: connection.connection_id,
+        type: 'auth',
+        originalActivityLogId: null,
+        environment_id: environment.id,
+        provider: config.provider
+    });
```

Comment:
I initially thought I was going to use this method so I refactored the signature